### PR TITLE
feat(jjb): Add file for app-service-configurable

### LINKED ISSUE #464: New Jenkins Pipeline needed for app-service-configurable
New Jenkins Pipeline needed for app-service-configurable
We'll need a pipeline job for the following repo.

https://github.com/edgexfoundry/app-service-configurable

It looks like all that's currently assigned is the DCO check. We'll want to treat this the same way we do device services, i.e. device-virtual-go, etc.

### PR DESCRIPTION
**Pull request by lenny-goodell**
Add jjb file for app-service-configurable using device-virtual file as a template.

closes #464

### CODE REVIEW

File: jjb/app-functions/app-service-configurable.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+
+- project:
+    name: app-service-configurable
+    project-name: app-service-configurable
+    project: app-service-configurable
+    mvn-settings: app-service-configurable-settings
+    pre_build_script: 'make test'
+    build_script: 'make build && make docker'
+    cron: 'H 11 * * *'
```
Comment (eb-oss) [inline jjb/app-functions/app-service-configurable.yaml]: Please remove this. We don't want all these jobs on a cron timer.

**Review by eb-oss — CHANGES_REQUESTED** [review]
In addition to remove the cron timer, please be sure to use the -s flag with git commit to add the signed-off-by line.

File: jjb/app-functions/app-service-configurable.yaml (hunk 1 above)
Comment (lenny-goodell) [reply jjb/app-functions/app-service-configurable.yaml]: Will do.  This was copied from device-virtual, so someone should look at why it is there and remove it if shouldn't be there.

**Review by jamesrgregg — CHANGES_REQUESTED** [review]
This needs to be a  Jenkins Pipeline job
This work was planned for this upcoming sprint and this service is slated to be used as a POC for the topic we discussed last week in DevOps WG.  Please do not merge this .

**Review by ernestojeda — CHANGES_REQUESTED** [review]
@lenny-intel I have a pipeline version of this already you can close this PR.

**Comment by lenny-goodell** [comment]
Closing per comments above.

**Pull request closed without merging**